project.clj

```diff
@@ -85,8 +99,11 @@
              :exclusions [org.clojure/clojure]]]                      ; TODO - should this be a dev dependency ?
   :main ^:skip-aot metabase.core
   :manifest {"Liquibase-Package" "liquibase.change,liquibase.changelog,liquibase.database,liquibase.parser,liquibase.precondition,liquibase.datatype,liquibase.serializer,liquibase.sqlgenerator,liquibase.executor,liquibase.snapshot,liquibase.logging,liquibase.diff,liquibase.structure,liquibase.structurecompare,liquibase.lockservice,liquibase.sdk,liquibase.ext"}
+  :java-source-paths ["src-java"]
   :target-path "target/%s"
   :jvm-opts ["-server"                                                ; Run JVM in server mode as opposed to client -- see http://stackoverflow.com/questions/198577/real-differences-between-java-server-and-java-client for a good explanation of this
+             "-Xmx4g"
+             "-Duser.timezone=Etc/UTC"
```

Comment:
what's the reasoning behind adding these settings? This would affect everyone who develops or runs a server via `lein`

Personally I prefer letting the JVM use as much memory as it wants on my dev machine. For example with 64 GB of RAM it often ends up using much more than 4GB. Most of the MB devs as well prefer sticking to our default system timezones of `US/Pacific` rather than UTC.